db/migrate/20200508211543_create_course_error_records.rb

```diff
@@ -0,0 +1,10 @@
+class CreateCourseErrorRecords < ActiveRecord::Migration[6.0]
+  def change
+    create_table :course_error_records do |t|
+      t.string :type_of_error
+      t.text :api_call_url
+
+      t.timestamps
```

Comment:
Can you put all these into one migration? For a new table with no existing data to worry about, doing all the changes at once makes it easier to make sense of.

Reply:
Done!